tutorials/transformations_02_coordinate_system_manager.ipynb

```diff
@@ -9,18 +9,6 @@
     "## Imports"
```

Comment:
<p>The overview you've just given, right? The link contains ALL the details (which might be overwhelming, if this is the starting point).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BAMWelDX/weldx/pull/767/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>It's more about the construction methods that aren't explicitly mentioned here. I think it is a managable task for the user to look through the API doc ;) </p>